File: Data Structures & Algorithms/duplicate-integer/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let number = false
        for(let n in nums){
            for(let j in nums){
                if(nums[n] == nums[j] && n != j){
                    number = true
                }
            }
        }
        return number
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-2.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        maxSize = float('-inf')
        for i in range(len(nums)):
            subtotal = 0
            for j in range(i, len(nums)):
                subtotal+= nums[j]
                maxSize = max(maxSize, subtotal)
        return maxSize